### ISSUE TITLE
Wysiwym: image upload modal click to upload unresponsive

### ISSUE THREAD

**Issue by kendallstrautman** [tinacms/tinacms#1405]
## Description

Upload image hint 'drag and drop or click to upload' bleeds off modal. And clicking on that upload image area doesn't open the local file upload screen. 

<img width="435" alt="Screen Shot 2020-08-12 at 2 13 24 PM" src="https://user-images.githubusercontent.com/36613477/90068908-51022b80-dca6-11ea-86cc-ae3c02101a5d.png">

cc @DirtyF 

### Steps to reproduce

Go to `demo-gatsby`, head to a block post. Hit the 'media icon' to see the style issue and try to click to upload from the filesystem.

**Comment by PCianes** [comment]
How do you make that icon appear to upload image within the Wysiwym editor?

**Comment by DirtyF** [comment]
@kendallstrautman you beat me to it, I'm not sure when this was introduced.

@PCianes Did you follow up our docs? https://tinacms.org/docs/ui/inline-editing/inline-wysiwyg#interface

**Comment by PCianes** [comment]
yes of course, and testing in tina-starter-grande I have not known how to implement it, so I was wondering if it was something not yet implemented

For example I'm not sure if the component InlineWysiwyg is injected from 'react-tinacms-editor' or 'react-tinacms-inline', and if it is necesary to work with 'react-markdown' or not

**Comment by DirtyF** [comment]
This also happens on tinacms.org, it's a regression as this was working before
<img width="899" alt="Capture d’écran 2020-08-17 à 15 29 51" src="https://user-images.githubusercontent.com/103008/90401920-f2babb80-e09e-11ea-906f-cd19513064e1.png">

Note: Drag and Drop works on tinacms.org but not on the Gatsby demo.

P.S: @PCianes Please go over to https://community.tinacms.org/ to ask about the implementation, we're discussing  a more specific issue here 😊

**Comment by PCianes** [comment]
thanks, i did it

![image](https://user-images.githubusercontent.com/29153003/90402988-96589b80-e0a0-11ea-93a9-4384ffe60ec6.png)

**Comment by DirtyF** [comment]
@PCianes Interesting the upload modal looks fine on Grande.
